isites_migration/management/commands/export_slide_tool.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+import mimetypes
+import json
+import ssl
+
+from optparse import make_option
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from boto.s3.connection import S3Connection
+from boto.s3.key import Key
+
+from icommons_common.models import (
+    Site, Topic, FileRepository, FileNode, FileNodeAttribute, ImageMetadata
+)
+
+
+logger = logging.getLogger(__name__)
+if hasattr(ssl, '_create_unverified_context'):
+    ssl._create_default_https_context = ssl._create_unverified_context
+
+
+class Command(BaseCommand):
+    help = 'Exports iSites Slide Tool topic file repositories to AWS S3'
```

Comment:
This command actually does only deal with exporting slide tool topics. Something else I had written to help out ATG from the isites_migration project that I am migrating over here.